docs/en/stores/basic-usage.md

```diff
@@ -0,0 +1,437 @@
+# Basic Usage
+
+Dojo provides a reactive architecture concerned with constantly modifying and rendering the current state of an application. In simple systems this can happen at a local level and widgets can maintain their own state. However, as a system becomes more complex the need to better compartmentalize and encapsulate data and create a clean separation of concerns quickly grows.
+
+Stores provide a clean interface for storing, modifying, and retrieving data from a global object through unidirectional data flow. Stores include support for common patterns such as asynchronous data retrieval, middleware, and undo. Stores and their patterns allow widgets to focus on their primary role of providing a visual representation of information and listening for user interactions.
+
+## The Store
+
+The store holds the global, atomic state for the entire application. The store gets created when the application gets created and defined in the `Registry` with an injector.
```

Comment:
Maybe say "The store should be created..." instead of "The store gets created...". I feel like the latter makes it sound like this will happen automatically.